magic_number: 0xA55A

version: 1

header_format: "!HBIH"

channels:
  - topic: "/test_udp"
    id: 0
    type: "rm_msgs/msg/TestRecv"
    direction: "uplink"